spec/assessments/textCompetingLinksSpec.js

```diff
@@ -0,0 +1,53 @@
+const TextCompetingLinksAssessment = require( "../../js/assessments/seo/textCompetingLinksAssessment.js" );
+const Paper = require( "../../js/values/Paper.js" );
+
+const factory = require( "../helpers/factory.js" );
+const i18n = factory.buildJed();
+
+describe( "An assessment for the text competing links", function() {
```

Comment:
"An assessment for competing links in the text"

Reply:
Fixed.